shell.nix

```diff
@@ -2,7 +2,7 @@
 # It just takes the shell attribute from default.nix.
 { config ? {}
 , sourcesOverride ? {}
-, withHoogle ? false
+, withHoogle ? true
```

Comment:
What's the effect of changing this to `true`?

Reply:
Makes `hoogle` command available in the shell and includes a hoogle index for all the packages that are needed by the project.